feat(ui): `:%detach` disconnects other UIs (#39216)
neovim/neovim: runtime/doc/news.txt

  • The |cmdwin-char| is shown via 'statuscolumn'.
• |gf| and |<cfile>| support `file://…` URIs.
• |:log| opens log files.
• |:restart| saves/restores the current session (window layout, buffers, …).
  • |:restart!| (with a bang "!") does not save/restore the session.
• |ZR| restarts Nvim (|:restart|).

> • |:%detach| detaches all other UIs, keeping only the current one.

Reviewer: ```suggestion
• |:detach| with range "%" detaches all UIs except the current one.
```
